pyupgrade/_plugins/native_literals.py

```diff
@@ -56,6 +74,9 @@ def visit_Call(
 ) -> Iterable[tuple[Offset, TokenFunc]]:
     if (
             state.settings.min_version >= (3,) and
-            is_a_native_literal_call(node, state.from_imports)
+            (
+                is_a_native_literal_call(node, state.from_imports) or
+                is_a_bytes_from_bytestr_call(node, state.from_imports)
+            )
     ):
-        yield ast_to_offset(node), _fix_native_str
+        yield ast_to_offset(node), _fix_native_literal
```

Comment:
I would add a separate `elif` for the bytestring check -- especially given the suggestion above